meta/template.md

```diff
@@ -1,48 +1,38 @@
 
-# Spec Name
+# Title: A human-readable title for this proposal
 
-* Status: In Review / Reviewed / Implementing / Implemented
-* Author(s): [Author Name](https://github.com/NuGet)
-* Issue: [1](https://github.com/NuGet/Home/issues/1) Issue Title
+[Author Name](https://github.com/<username>) | [GitHub Issue](https://github.com/NuGet/Home/issues/<issue>)
 
-## Problem Background
+## Description: What is it?
 
-Present the background and the motivation for this design. 
-Present evidence why this is an important problem to tackle.
+A concise, one-paragraph description of the proposal.
 
-## Who are the customers
+## Problem: What problem is this solving?
 
-Customers affected by this change.
+A short background of the problem and how your solution will solve it.
 
-## Goals
+## Why: How do we know this is a real problem and worth solving?
 
-Specifically call out the goals for this proposed design.
+A few details on why we're doing this, and what pain points this solves. Use real, concrete examples to make your case.
 
-## Non-Goals
+## Success: How do we know if we've solved this problem?
 
-Call out things that are not in consideration for this design. 
+A couple ways on how we might measure the success or outcome of this proposal. 
 
-## Solution Overview
+## Audience: Who are we building for?
 
-Describe the proposed solution in as much detail as you deem necessary.
+A brief description to help everyone understand who this proposal is for.
 
-## Test Strategy
+## What: What does this look like in the product?
 
-Other than Unit Tests, which automated and manual integration, component, and end-to-end tests will be needed to validate the implementation? Is a special roll-out strategy (including kill-switch, opt-in flag, etc.) needed?
+A low-fidelity mockup or pseduocode that shows how this proposal would look.
 
-## Future Work
+## How: How should it be implemented?
 
-Declare any future work if applicable.
+A design or implementation for bringing the proposal to life. 
 
-## Open Questions
+## When: When does this ship and what are the milestones?
 
-Any open questions not specifically called out in the design.
+A timeline including ideal dates to ship the proposal & any key milestones (if any).
```

Comment:
This proposal process is open to third-parties, right? How will they determine ship dates? Even for folks on the NuGet team, this seems like a hard section to fill out as part of a proposal. What do you think of removing this section?

Reply:
Great point. I think we should remove it then.